sources/personio/README.md

```diff
@@ -27,22 +28,33 @@ dlt init personio duckdb
 Here, we chose duckdb as the destination. Alternatively, you can also choose redshift, bigquery, or
 any of the other [destinations](https://dlthub.com/docs/dlt-ecosystem/destinations/).
 
-## Add credentials
+## Grab Personio credentials
```

Comment:
use` ## Setup verified source` instead of `## Grab Personio credentials`, for unification

Reply:
done